pegasus/sites.v3/code.org/public/js/workshop-search.js

```diff
@@ -35,7 +42,7 @@ function loadWorkshops() {
 
         var infowindow_content = '';
 
-        if(typeof markers[hash]==='undefined'){
+        if (typeof markers[hash]==='undefined') {
```

Comment:
I cleaned up this code the minimum needed to get it to pass the linting rules and function properly. It's not needed, but I prefer it as well so I'll add them. In general this code can be cleaned up quite a bit.